AISKU/Tests/ApplicationInsightsDeprecatedTests.ts

```diff
@@ -1,7 +1,5 @@
 import { IAppInsightsDeprecated } from "../src/ApplicationInsightsDeprecated";
-import { ApplicationInsightsContainer } from "../src/ApplicationInsightsContainer";
-import { Snippet } from "../src/Initialization";
-import { Sender } from "@microsoft/applicationinsights-channel-js";
+import { ApplicationInsightsContainer, Snippet, Sender } from "@microsoft/applicationinsights-web";
```

Comment:
Changed the tests to consume the module rather than using the source code to try and repro the failure :-) It didn't, but this is will capture issues in the future where we don't export a class / code.

Reply:
Removing this change